src/Engine/System/RenderSystem.cpp

```diff
@@ -78,6 +85,21 @@ void RenderSystem::update(float dt) {
         ImGui::Render();
```

Comment:
ImGui shouldn't be included in post-processing. This #ifdef to #endif should be moved to after any post-processing is complete

Reply:
Ah yeah lol it would suck to see a blurry imgui

Reply:
Got it, will change